tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -12628,11 +12629,30 @@ gen_acon_expr(int ilix, LL_Type *expected_type)
   sym_is_refd(sptr);
   idx = (STYPEG(sptr) == ST_STRUCT || STYPEG(sptr) == ST_ARRAY
          || ACONOFFG(opnd) < 0) ? 0 : ACONOFFG(opnd);
-  process_sptr_offset(sptr, variable_offset_in_aggregate(sptr, idx));
+  if (DTA(dtype)) {
```

Comment:
NIT: Please format the whole `if`-`else` statement with clang-format. Capitalize and punctuate the comment.

Reply:
It may be clearer to rewrite like this (following the example of the code below):
```c
  /* Handle !dir$ align pragma. Adjust sptr's offset in ag to be aligned with
   * sptr's alignment.
   */
  if (DTA(dtype))
    idx += ALIGN(variable_offset_in_aggregate(sptr, 0), DTA(dtype));
  else
    idx = variable_offset_in_aggregate(sptr, idx);
  process_sptr_offset(sptr, idx);
```

Reply:
> It may be clearer to rewrite like this (following the example of the code below):
> 
> ```c
>   /* Handle !dir$ align pragma. Adjust sptr's offset in ag to be aligned with
>    * sptr's alignment.
>    */
>   if (DTA(dtype))
>     idx += ALIGN(variable_offset_in_aggregate(sptr, 0), DTA(dtype));
>   else
>     idx = variable_offset_in_aggregate(sptr, idx);
>   process_sptr_offset(sptr, idx);
> ```

Thanks for you suggestion.
This part is no longer needed in the latest patches.